pulpcore/cli/common/context.py

```diff
@@ -1,833 +1,23 @@
-import datetime
-import re
-import sys
-import time
-from typing import Any, ClassVar, Dict, List, Mapping, NamedTuple, Optional, Set, Type, Union
-
-from packaging.version import parse as parse_version
-from requests import HTTPError
-
-from pulpcore.cli.common.i18n import get_translation
-from pulpcore.cli.common.openapi import OpenAPI, OpenAPIError, UploadsMap
-
-translation = get_translation(__name__)
-_ = translation.gettext
-
-DEFAULT_LIMIT = 25
-BATCH_SIZE = 25
-DATETIME_FORMATS = [
-    "%Y-%m-%dT%H:%M:%S.%fZ",  # Pulp format
-    "%Y-%m-%d",  # intl. format
-    "%Y-%m-%d %H:%M:%S",  # ... with time
-    "%Y-%m-%dT%H:%M:%S",  # ... with time and T as a separator
-    "%y/%m/%d",  # US format
-    "%y/%m/%d %h:%M:%S %p",  # ... with time
-    "%y/%m/%dT%h:%M:%S %p",  # ... with time and T as a separator
-    "%y/%m/%d %H:%M:%S",  # ... with time 24h
-    "%y/%m/%dT%H:%M:%S",  # ... with time 24h and T as a separator
-    "%x",  # local format
-    "%x %X",  # ... with time
-    "%xT%X",  # ... with time and T as a separator
+from pulp_glue.common.context import (  # noqa: F401
```

Comment:
This is so that any cli-plugins we don't know about, will continue working post-this-change, correct?

Reply:
Absolutey!